x-pack/plugins/security_solution/public/common/components/event_details/table/use_action_cell_data_provider.ts

```diff
@@ -50,72 +66,90 @@ export const useActionCellDataProvider = ({
   isObjectArray,
   linkValue,
   values,
-}: UseActionCellDataProvider): { idList: string[]; stringValues: string[] } | null => {
-  if (values === null || values === undefined) return null;
-
-  const stringifiedValues: string[] = [];
-  const arrayValues = Array.isArray(values) ? values : [values];
+}: UseActionCellDataProvider): {
+  stringValues: string[];
+  dataProvider: DataProvider[];
+} | null => {
+  const cellData = useMemo(() => {
+    if (values === null || values === undefined) return null;
+    const arrayValues = Array.isArray(values) ? values : [values];
+    return arrayValues.reduce<{
+      stringValues: string[];
+      dataProvider: DataProvider[];
+    }>(
+      (memo, value, index) => {
+        let id: string = '';
+        let valueAsString: string = isString(value) ? value : `${values}`;
+        const appendedUniqueId = `${contextId}-${eventId}-${field}-${index}-${value}`;
+        if (fieldFromBrowserField == null) {
+          memo.stringValues.push(valueAsString);
+          return memo;
+        }
 
-  const idList: string[] = arrayValues.reduce((memo, value, index) => {
-    let id = null;
-    let valueAsString: string = isString(value) ? value : `${values}`;
-    if (fieldFromBrowserField == null) {
-      stringifiedValues.push(valueAsString);
-      return memo;
-    }
-    const appendedUniqueId = `${contextId}-${eventId}-${field}-${index}-${value}-${eventId}-${field}-${value}`;
-    if (isObjectArray || fieldType === GEO_FIELD_TYPE || [MESSAGE_FIELD_NAME].includes(field)) {
-      stringifiedValues.push(valueAsString);
-      return memo;
-    } else if (fieldType === IP_FIELD_TYPE) {
-      id = `formatted-ip-data-provider-${contextId}-${field}-${value}-${eventId}`;
-      if (isString(value) && !isEmpty(value)) {
-        try {
-          const addresses = JSON.parse(value);
-          if (isArray(addresses)) {
-            valueAsString = addresses.join(',');
+        if (isObjectArray || fieldType === GEO_FIELD_TYPE || [MESSAGE_FIELD_NAME].includes(field)) {
+          memo.stringValues.push(valueAsString);
+          return memo;
+        } else if (fieldType === IP_FIELD_TYPE) {
+          id = `formatted-ip-data-provider-${contextId}-${field}-${value}-${eventId}`;
```

Comment:
My bad here 😅 , I can create a separate ticket for this, but I think we should use this or some common function like this to make the id's within the `formatted_field` component as well so we don't end up out of sync by accident

Reply:
I tried to not touch it too much here, but I do not think that we need the id as much as before since we are not using the draggable but I kept it just in case if we need to bring it back